.github/workflows/ci.yml

```diff
@@ -13,24 +13,24 @@ env:
   UGBIO_BASE_IMAGE_NAME: ugbio_base:1.2.0
 
 jobs:
-  get-workspaces:
+  get-members:
     runs-on: ubuntu-latest
     outputs:
-      matrix: ${{steps.list_dirs.outputs.workspaces}}
+      matrix: ${{steps.list_members.outputs.members}}
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/actions/get-workspaces
-        id: list_dirs
+      - uses: ./.github/actions/get-members
```

Comment:
Not critical, but to me it looks nicer and clearer to call directly the python script
```
matrix=$(python3 ${GITHUB_WORKSPACE}/.github/utils/ugbio_members_matrix.py ${keep_internal})
      {
        echo 'matrix<<EOF'
        echo ${matrix}
        echo EOF
      } >> ${GITHUB_OUTPUT}
```